src/components/TabMenu/Item/style.css

```diff
@@ -8,6 +8,14 @@
   box-sizing: content-box;
 }
 
+.Item:first-of-type {
```

Comment:
I know the linter warns about this, but it's not exactly the same thing.
If you need to support ...
```jsx
<span /><Item />
```
... and Item renders a div, it will break; because Item is still the first of its type

May not be an issue for tabs, but worth checking.

Reply:
I'm not totally worried about any other divs, because if the behavior is consistent then the first <span /> or whatever can apply its own spacing. Is that a reasonable call to make?

Reply:
Works for me.